cf/data/data.py

```diff
@@ -30,14 +30,14 @@
     _numpy_isclose,
     _section,
     abspath,
+    atol,
+    default_netCDF_fillvals,
+    fm_threshold,
+    free_memory,
+    log_level,
+    parse_indices,
+    rtol,
 )
-from ..functions import atol as cf_atol
-from ..functions import default_netCDF_fillvals
-from ..functions import fm_threshold as cf_fm_threshold
```

Comment:
Hi - I think this is OK to let this through, as the `in_memory` method has been moved to `deprecated.py` in another open PR (#387), and I wanted to avoid a merge conflict by modifying a modified line.

Reply:
Ah I see, thanks for the context! That's fine then, I'll mark this as resolved.